cmd/get_cluster.go

```diff
@@ -17,7 +21,59 @@ var getClusterCmd = &cobra.Command{
 	Short: "list or get clusters",
 	Long:  `list or get cluster in mystack`,
 	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Println("get_cluster called")
+		ll := logrus.InfoLevel
```

Comment:
this same code until line 40 is replicated in all commands, maybe we should make a helper method to avoid duplicating code?